anonlink/entitymatch.py

```diff
@@ -11,6 +13,17 @@
 log = logging.getLogger('anonlink.entitymatch')
 
 
+def _fname():
+    return inspect.currentframe().f_back.f_code.co_name
+def _deprecation(use_instead=None):
+    msg = (f'anonlink.entitymatch.{_fname()} has been deprecated ')
+    if use_instead is not None:
+        msg += '(use anonlink.{use_instead} instead)'
```

Comment:
Doesn't that need to be an f-string?

I nearly didn't bother looking at this line because it appears to be copy pasted... which is one strong reason not to repeat yourself.